.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 100
```

Comment:
If we want to change it to `100`, please update the `rustfmt.toml` also. There's a `max_width` set to 120, and we probably want those 2 values to be the same

Reply:
that's because rustfmt's default is [`max_width=100`](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=width#max_width)

I would rather delete rustfmt.toml than tune it TBH, wdyt?

Reply:
Imho it would be best to remove the `max_width` from `rustfmt.toml`, and let it set it to the default value (100). If you meant deleting whole `rustfmt.toml` then I'm not so sure about that. I would opt for changing it and adjusting it to our needs rather than deleting the whole file.

Reply:
Ok, after thinking about that, I think it would be the best to remove the `rustfmt.toml` completly as for now

Reply:
Yeah, what I mean is to adjust our needs to rustfmt defaults entirely :D